content/fr/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -676,7 +676,7 @@ si le master est irrécupérable, votre cluster peut perdre ses données et peut
 partir de zéro. L'ajout du support HA (plusieurs serveurs etcd, plusieurs API servers, etc.)
 à kubeadm est encore en cours de developpement.
 
-   Contournement: régulièrement [sauvegarder etcd](https://coreos.com/etcd/docs/latest/admin_guide.html).
+   Contournement: régulièrement [sauvegarder etcd](https://etcd.io/docs/v3.5/op-guide/recovery/).
```

Comment:
```suggestion
   Contournement: régulièrement [sauvegarder etcd](https://etcd.io/docs/latest/op-guide/recovery/).
```

Reply:
Please don't choose a specific version

Reply:
I don't mind, but I need to change it in the English version, I'm just doing a little synchronization